nebula_ros/launch/continental_launch_all_hw.xml

```diff
@@ -24,7 +29,8 @@
     <group if="$(eval &quot;'$(var sensor_model)' == 'SRR520' &quot;)" >
 
         <node pkg="nebula_ros" exec="continental_srr520_ros_wrapper_node" name="nebula_continental_srr520" output="screen">
-	    <param from="$(var config_file)" allow_substs="true"/>
```

Comment:
There seems to be a missalign in this line :sweat_smile:

Reply:
Ah, that misaligned line was removed in this PR. The corrected line is line `32` below :raised_hands: